visualization/income_edu/income.html

```diff
@@ -42,14 +42,16 @@
 			// 1. Filter data: Keep only people who:
 			// 	- were employed with an organization (as opposed to self-employed)
 			data = data.filter(person => +person["INCWAGE"] > 0);
-			// - TODO: I'm currently filtering out people with incomes > $200,000, when what I probably want to do 
-			// is either top-code them or handle this piece of code directly in the visualization.
-			data = data.filter(person => +person["INCWAGE"] <= 200000);
 			// 	- worked the entire year (50+ weeks)
 			data = data.filter(person => +person["WKSWORK1"] >= 50);
 			// 	- usually worked full time (35+ hours/week)
 			data = data.filter(person => +person["UHRSWORKLY"] >= 35);
-			
+			// - Top-code people with incomes > $200,000.
```

Comment:
I have mixed feelings about this - topcoded data can give a false reading of the data that is more surprising than omitting. It might be worth visually separating it from the remainder of the data so it's clear how much data is in the top-code category

Reply:
I'm trying to brainstorm how I could visually separate the top-coded data: different color (darker?), different shape (e.g. square instead of circle), different size (bigger?), above the $200,000 points, maybe separated by a dotted horizontal line or vertical "..."s? Not sure I have a workable solution yet. Any thoughts, or shall I just leave it for now and keep an eye out for ideas?